.gitignore

```diff
@@ -1,5 +1,8 @@
+<<<<<<< HEAD
+=======
 models/
 *.csv
+>>>>>>> a1b4dead406f8d65e91a5270570707fecb286961
```

Comment:
Why is this `Why is this `<<<<<<<<<< HEAD` committed to the source code?` committed to the source code?